config/environments/development.rb

```diff
@@ -77,4 +77,12 @@
       },
     }
   )
+
+  config.after_initialize do
```

Comment:
Could we maybe have `Bullet.add_footer` here? Might make it a bit clearer that something needs fixing when devs are testing in development

Reply:
I'm wary of gem code that inserts markup or JS into the page as it can result in weird behaviour. Also because of the circular false positives it will literally appear on half the claim pages, complaining about missing eager-loading of `eligibility`, `claim_school`, and/or `current_school`. Short of doing something where we conditionally eager load the appropriate association on each page depending on what is used, I don't think there is a way to make it so we don't end up ignoring these anyway because of all the false positives. Also, that eager loading would be mostly pointless, because when you call `TslrClaim.includes(:eligibility).find(ID)`, Rails query optimiser decides it's not worth the join and just makes two SQL queries anyway!

Reply:
Yeah, I've felt that pain before

Reply:
Been thinking about this more and maybe it's better to have this stuff in test (maybe with `rails_logger` left out?), so if there are any issues with performance we can look at the bullet logs? As our test coverage is pretty good, we can be confident that a run of the whole test suite is covering pretty much all of the queries, and by scanning the Bullet logs we can see any new N+1 queries.

Reply:
Yep, ok PR updated.